[ADD] Live chat: participate (#9958)
odoo/documentation: content/applications/websites/livechat/participate.rst

========================
Participate in live chat
========================

*Operators* are the users who respond to live chat requests from customers and website visitors. The
information below outlines the necessary steps for operators participating in live chat
conversations on an Odoo database.

Set an online chat name
-----------------------

Before participating in a live chat, operators may choose to update their *Online Chat Name*. This

> is the name that is displayed to site visitors in the live chat conversation.

Reviewer: ```suggestion
is the name displayed to site visitors in the live chat conversation.
```

> To update the :guilabel:`Online Chat Name`, click on the user avatar in the top-right corner of any

Reviewer 2: ```suggestion
To update the *Online Chat Name*, click on the user avatar in the top-right corner of any
```

page in the database. Select :guilabel:`My Profile` from the drop-down menu to open the profile
page. On the right side of the :guilabel:`Preferences` tab, locate the :guilabel:`Online Chat Name`
field, and enter a preferred name.

If a user's :guilabel:`Online Chat Name` is not set, the name displayed defaults to the user's name
that is designated on their profile page.

.. example::
   A user has their full name as their :guilabel:`User Name`, but they do not want to include their
   last name in a live chat conversation. They would then set their :guilabel:`Online Chat Name` to
   include only their first name.

   .. image:: participate/online-chat-name.png
      :align: center
      :alt: View of user profile in Odoo, emphasizing the Online Chat name field.

Set online chat languages
-------------------------

If an operator speaks multiple languages, they can add this information to their user profile. This

> information can then be used to determine what conversations the operator is assigned. A visitor's

Reviewer: ```suggestion
information can then be used to determine which conversations the operator is assigned. A visitor's
```

> language is determined via their browser's language settings.

Reviewer 2: ```suggestion
language is determined, via their browser's language settings.
```

.. note::

>    Conversations are assigned to operators based on a number of criteria, including availability and

Reviewer 2: ```suggestion
   Conversations are assigned to operators, based on a number of criteria, including availability and
```

   the number of ongoing conversations. While the operator's main language and additional languages

>    are taken into consideration, they do **not** supersede all other criteria.

Reviewer: Are there any criteria that they do supersede?

right side of the :guilabel:`Preferences` tab, click into the :guilabel:`Online Chat Language`
field, and select one or more languages from the drop-down menu.

.. important::
   Only :doc:`languages <../../general/users/language>` that are enabled on the database can be
   selected in the :guilabel:`Online Chat Language` field.

Join or leave a channel
-----------------------

To join a live chat channel, go to the :menuselection:`Live Chat app --> Channels`, and click the
:guilabel:`Join` button on the Kanban card for the appropriate channel.

Any channel where the user is currently active shows a :guilabel:`Leave` button. Click this button
to disconnect from the channel.

.. image:: participate/leave-channel.png
   :align: center
   :alt: View of a channel form and the option to join a channel for Odoo Live Chat.

.. important::
   Operators that do not show any activity in Odoo for more than thirty minutes are considered
   disconnected, and subsequently removed from the channel.

Manage live chat requests
-------------------------

When an operator is active in a channel, chat windows open in the bottom-right corner of the

> screen, no matter where they are in the database. They can participate in conversations without
> leaving their current location.

Reviewer: ```suggestion
screen, no matter where they are in the database. This allows them to access other pages and apps, while still participating in the conversation.
```

.. tip::
   Conversations can also be accessed by clicking the :icon:`fa-comments` :guilabel:`(messages)`
   icon in the menu bar.

   .. image:: participate/menu-bar.png
      :align: center
      :alt: View of the menu bar in Odoo emphasizing the comments icon.

> Live chat conversations can also be viewed by navigating to :menuselection:`Discuss app`. New

Reviewer: ```suggestion
Live chat conversations can also be viewed by navigating to the :menuselection:`Discuss app`. New
```
